load-testing/tests/relays_stress.feature

```diff
@@ -10,8 +10,8 @@ Feature: Loading gateway server with relays
       | supplier    | 1     |
     And more actors are staked as follows:
       | actor       | actor inc amount | blocks per inc | max actors |
-      | application | 4                | 4              | 12         |
-      | gateway     | 1                | 4              | 3          |
-      | supplier    | 1                | 4              | 3          |
+      | application | 4                | 10             | 12         |
+      | gateway     | 1                | 10             | 3          |
+      | supplier    | 1                | 10             | 3          |
```

Comment:
I was under the impression that the "blocks per inc" needed to be a multiple of the blocks per session param to maintain constant rates of change across various metrics as the test scales actors. I also thought that there was a check for this somewhere in the load test helpers, around the "plans".

Reply:
@bryanchriswhite Are you sure about this?

1. If so -> can you help @okdas look into the code and find the reasoning.
2. If not -> my intuition is that it keeps things simpler (to reason about) but doesn't need to be enforced.

In real life, we'll be staking / unstaking irrespective of the blocker per session so it doesn't make sense for the framework to have this limitation.

@okdas W/e the resolution ends up being, seems like the helper in the code needs a #PUC.

Reply:
@bryanchriswhite interesting. [This check](https://github.com/pokt-network/poktroll/blob/main/load-testing/tests/relays_stress_helpers_test.go#L341) is the reason I adjusted the number - as I was getting an error. Given our current blocks per session [is 10](https://github.com/pokt-network/poktroll/blob/main/config.yml#L242) ([bumped from 4 a couple of weeks ago](https://github.com/pokt-network/poktroll/commit/738e3429c06d7752cb3467b6b90c2710718700a7#diff-5a7db8dbadaef1b1b5a8738ba70b5ffac82b8e243732154165911284e08aad4bL202)), what do you think should be the best value?

Reply:
@okdas that's the check I was referring to. :+1: I did not realize that blocks per session was 10; this all makes sense now.